### PR TITLE
Add deployment recommendations example notebook

### PR DESCRIPTION
**Pull request by gwang111**
*Issue #, if available:*

*Description of changes:*
Add an example notebook demonstrating the new Deployment Recommendations feature in DescribeModel

*Testing done:*
E2E Notebook Test

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: sagemaker-inference-recommender/deployment-recommendations/instant-recommendations.ipynb [hunk 1]
```diff
@@ -0,0 +1,367 @@
+{
```
Comment (atqy) [inline sagemaker-inference-recommender/deployment-recommendations/instant-recommendations.ipynb:1]: <p>Please clear output to keep notebook clean</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4077/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (atqy) [inline sagemaker-inference-recommender/deployment-recommendations/instant-recommendations.ipynb:1]: <p>Where does the model come from? Is it an empty model? If not, does it have legal approval?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4077/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (gwang111) [reply sagemaker-inference-recommender/deployment-recommendations/instant-recommendations.ipynb:1]: <p>hey yeah its just an empty model that takes in a sagemaker dlc. thats it</p>

Comment (gwang111) [reply sagemaker-inference-recommender/deployment-recommendations/instant-recommendations.ipynb:1]: <p>ack</p>

**Review by atqy — APPROVED** [review]

**Pull request merged**